workspace/api/openfdd_bridge/modbus_store.py

```
@@ -0,0 +1,510 @@
+                }
+            )
+            ingest = append_samples_and_ingest(
+                host=host,
+                unit_id=unit_id,
+                readings=result.get("readings") or [],
+            )
+            if ingest.get("ok"):
+                total_samples += int(ingest.get("samples_appended") or 0)
+            for row in group:
+                _POLL_LAST_RUN[str(row.get("point_id") or "")] = now
+        except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)